backend/data/src/main/resources/db/changelog/schema.xml

```diff
@@ -2699,4 +2699,27 @@
     <changeSet author="stepangranat (generated)" id="1692866751704-1">
         <dropDefaultValue columnDataType="boolean" columnName="hidden" tableName="tolgee_batch_job"/>
     </changeSet>
+    <changeSet author="cynthia (generated)" id="1693517732527-1">
+        <addColumn tableName="user_account">
+            <column name="is_initial_user" type="boolean" defaultValue="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cynthia" id="1693517732527-2">
+        <sql dbms="postgresql">
+            UPDATE user_account
```

Comment:
Postgres doesn't return results in any order by default. So when this is executed on Cloud instance, it selects random user and sets it initial user.

Reply:
> Doesn't it implicitly have insert-order ordering... 

I don't think so.

Reply:
I still really am not sure what needs to be fixed here ;-;

The WHERE clause narrows it to the first entry ordered by ID ascending excluding the implicit user (or at least I do think it does that)

Reply:
There is no default order in Postgres. This will just take random user and sets it initial, so at minimal id should work fine. `SELECT min(id) FROM user_account group by id`

Reply:
The request I'm using does use an `ORDER BY id`, isn't this enough?

`SELECT id FROM user_account WHERE username != '___implicit_user' ORDER BY id LIMIT 1`